demo_working_hybrid.py

```
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)